.github/ISSUE_TEMPLATE/unmaintained.yaml

```diff
@@ -0,0 +1,289 @@
+name: Unmaintained component
+description: Create an issue to track the deprecation of a component
+labels: ["help wanted", "unmaintained"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please open an issue using this template if code owners for a component have been unresponsive in 6 weeks. Ensure
+        links to relevant pull requests and issues are added to this issue.
```

Comment:
Would it make sense to add a specific section for relevant links?

Reply:
Added a section at the bottom of the template